apps/frontend/src/libs/modules/http/base-http.module.ts

```diff
@@ -0,0 +1,15 @@
+import { type HTTP, type HTTPOptions } from "./libs/types/types.ts";
+
+class BaseHttp implements HTTP {
```

Comment:
```suggestion
import { type HTTP, type HTTPOptions } from "./libs/types/types.ts";

class BaseHttp implements HTTP {
```
There are two approaches to writing HTTP in the application - HTTP or Http.
please select only one and change it throughout the application